ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -6,6 +6,7 @@ set -o pipefail
 
 # TODO: move to image
 curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o /tmp/jq && chmod +x /tmp/jq
```

Comment:
If I remember right, jq is already installed in installer pod, right?

Reply:
yes, `jq` is in the original image:

```
  from_image:
    namespace: ocp
    name: "4.5"
    tag: upi-installer
```

but I changed it to following image due to newer version of `openshift-install` is required to run command `openshift-install coreos print-stream-json`:
https://github.com/openshift/release/blob/c5b323a84c4a345325d0f7b58bb0d4bebad77f6c/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml#L1-L5

but `jq` does not exist in this image.